nirc_ehr/resources/queries/study/demographicsLastProject.sql

```diff
@@ -0,0 +1,10 @@
+
+SELECT
+    d2.id,
+    d2.project,
+    h.enddate as enddate
+
+FROM study.assignment d2
+         JOIN (SELECT id, max(date) as maxDate, max(enddate) as enddate FROM study.assignment h GROUP BY id) h
+              ON (h.id = d2.id and d2.date = h.maxdate)
+WHERE d2.qcstate.publicdata = true
```

Comment:
Is this supposed to be the previous assignment before it got reassigned to the current one? What if there is no previous assignment, ok with displaying the current one?

Reply:
ok good to know!